orchagent/orch.cpp

```diff
@@ -30,6 +30,20 @@ Orch::Orch(DBConnector *db, const vector<string> &tableNames)
     }
 }
 
+Orch::Orch(swss::DBConnector *db1, swss::DBConnector *db2, 
```

Comment:
What is this change? Seems not related to feature. Please remove

Reply:
This change is needed, as Vlanmgr needs to process updates from 2 db's now, config db and state db.

Reply:
Please explain which tables. and where is it added in Vlanmgr code?

Reply:
Currently, Vlanmgr processes updates only from the config db. PAC sends updates to Vlanmgr through the following tables in the state db:
#define STATE_OPER_VLAN_TABLE_NAME "OPER_VLAN"
#define STATE_OPER_VLAN_MEMBER_TABLE_NAME "OPER_VLAN_MEMBER"
#define STATE_OPER_FDB_TABLE_NAME "OPER_FDB"
#define STATE_OPER_PORT_TABLE_NAME "OPER_PORT"

To process these updates, Vlanmgr needs to add the state db tables as well to the Orch list:

VlanMgr::VlanMgr(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, const vector<string> &tableNames,
        const vector<string> &stateTableNames) :
        Orch(cfgDb, stateDb, tableNames, stateTableNames),

So, in addition to cfgDb, we also add stateDb in the Orch list. The PAC related tables are in stateTableNames

Reply:
Why is this not coming via APP_DB? I don't think we have a current flow where orchagent listens to STATE_DB.

Reply:
Orchagent does not listen to STATE_DB. As per the PAC design in the HLD, Vlanmgr listens to STATE_DB for updates from PAC. Vlanmgr uses Orch infra for subscription.

Reply:
HLD 3.1.2 has the call flow, vlanmgr will subscribe and publish to state_db, it will also update to appl_db. No change to existing OA pub/sub model with PAC, OA will subscribe to appl_db for PAC update and publish to asic_db